curator/actions.py

```diff
@@ -2091,17 +2091,24 @@ def __init__(
         self.number_of_shards = number_of_shards
         self.wait_for_active_shards = wait_for_active_shards
         self.shrink_node_name = None
+
         self.body = {
             'settings': {
                 'index.number_of_shards' : number_of_shards,
                 'index.number_of_replicas' : number_of_replicas,
-                'index.routing.allocation.require._name': None,
-                'index.blocks.write': None
             }
         }
+
         if extra_settings:
             self._merge_extra_settings(extra_settings)
 
+        if utils.get_version(self.client) > (6, 0, 1):
```

Comment:
Please make this `>= (6, 1, 0)` as all of the 6.0.x versions will have the API difference.

Reply:
Done